docs/modules/manual/pages/objecten.adoc

```diff
@@ -1,5 +1,84 @@
-= Objecten
-:description: Een omschrijving van de architectuur van het project.
+= Hoe beheer ik de objecten van mijn erfgoedinstelling?
+:description: Een gebruikershandleiding voor het beheren van objecten in Solid CRS.
```

Comment:
For the whole document: perhaps to replace "commands" with suggestions.

E.g. `Ga naar het object dat je wilt verwijderen. Druk in de rechterbovenhoek op het vuilbak-icoontje. Het object is nu verwijderd.` => `Een object verwijderen uit een collectie kan via het vuilbak-icoontje rechtsboven op de Object-overzichtspagina. Het object is nu verwijderd.`